docs/src/main/sphinx/connector/cassandra.rst

```diff
@@ -8,12 +8,12 @@ The Cassandra connector allows querying data stored in
 Requirements
 ------------
 
-Requirements for using the connector in a catalog to connect to a Cassandra
-data source are:
+To connect to a Cassandra data source using the connector in a catalog, you
+need:
 
-* Cassandra version 2.2 or higher
-* Network access, by default on port 9042, from the Trino coordinator and
-  workers to Cassandra.
+* Cassandra version 2.2 or higher.
+* Network access from the Trino coordinator and workers to Cassandra.
+  Port 9042 is the default port and is used in the code examples.
```

Comment:
Not sure why this appears to be indented

Reply:
Here and all others ... this is correct as the sentence is part of the same bullet item.